azkaban-db/src/main/sql/create.execution_flows.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE execution_flows (
   flow_data   LONGBLOB,
   executor_id INT                   DEFAULT NULL,
   use_executor INT                  DEFAULT NULL,
+  flow_priority INT                 DEFAULT 5,
```

Comment:
consider enum type? Then priority would be limited instead of any integer.

Reply:
MySQL ENUM type? I think it will artificially limit what we can do with priorities in the future or make it hard to maintain at least. There have been comments around setting priorities based on groups. For example: groupA could set priorities from 1-10, groupB from 11-20, etc

Reply:
discussed offline, we agree to it's better to have a fixed shorter range for priority and 
enforce flow priority range limitation on application side.